content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,206 @@
+
+By using Nightly, you will help us make a really great Firefox and as
+a benefit you will have this great Firefox experience 3 months ahead
+of schedule!
+
+Also most of the time, if you discover a problem with your Firefox
+Nightly, it will be fixed in the next release the day after, while
+with Firefox Release you will potentially have to wait for 6 weeks.
+
+Help add-ons developer to validate their add-ons is still working
+-----------------------------------------------------------------
+
+Firefox is transitionning from a complex and error prone way of
+writting add-ons to a standardized way of writting web extensions,
+which are most likely to be compatible with Chrome and use well known
+web technologies which makes them simple to get started with.
+
+This is challenging for add-ons creators that already have a working
+add-ons that needs to be rewritten.
+
+As a community we can help them. If they already have a Google Chrome
+add-ons it is most likely that they will be able to install it in
+Firefox directly but Mozilla Firefox also expose more add-ons API than
+Chrome and we will create more in 2017 to help as much as possible
+port previous add-ons to the web extension standard.
+
+As a user we can help to test this new add-ons to keep a good
+experience for our users.
+
+Have a better developer experience to develop your web apps
+-----------------------------------------------------------
+
+If you develop web applications you might have had some problems to
+debug them with Firefox.
+
+We have a full team of developers dedicated to make Firefox the best
+tool for developping web applications. This team is improving web
+developer tools everyday and by using Nightly it will make your life
+easier.
+
+I wish that at some point you would even want to open Firefox because
+it is easiser than debugging your app in Chrome. And I have good hope
+that we reach this point in 2017.
+
+
+Give a really valuable contribution to the Mozilla project without coding
+-------------------------------------------------------------------------
+
+Often people ask me how they can contribute to the Mozilla Firefox
+project.
+
+To be honest, starting to use Nightly today is a real huge step toward
+that goal.
+
+To use Firefox Nightly as a daily user is a simple and awesome way to
+help Firefox because it gives us feedback on the work we are doing
+ahead of time and also everyday. We don't have to wait for 18 weeks to
+get feedback on our work, we can have it the day after.
+
+By the time I write this article we have about 35000 daily users of Firefox
+Nightly so your choice to use it makes a huge difference to us.
+
+
+Conclusion
+----------
+
+I hope you are now convinced that you can help us to make a better
+Firefox by starting to use Firefox Nightly today.
+
+You can install the `desktop Firefox version <https://nightly.mozilla.org/>` as well as your
+`Android Firefox version <https://www.mozilla.org/fr/firefox/channel/android/>`
```

Comment:
use https://www.mozilla.org/firefox/channel/android/ to have language negociation handled by mozilla.org